mission_systems/shooter/nodes/Sabertooth2x12.py

```diff
@@ -11,6 +11,9 @@ def __init__(self,filename,sim=False):
         self.sim = sim
         if not self.sim:
             self.ser = serial.Serial(filename,baudrate=19200)
+        else:
+            global rospy
+            import rospy       
```

Comment:
Usually people will have two files, the actual driver node and then the ros plugin for the driver that uses the actual driver node's API.